contracts/interfaces/IAxelarAmplifierGatewayAuth.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+interface IAxelarAmplifierGatewayAuth {
```

Comment:
rename to `IAxelarAmplifierAuth`?

Reply:
That already exists, this is the generic auth interface

Reply:
Maybe call the generic one `IAmplifierAuth` ?

Reply:
The interface requirement is specific to the gateway, so I feel that IAmplifierAuth is a bit too generic, and there's no meaningful difference between AxelarAmplifier and Amplifier